package-lock.json

```diff
@@ -0,0 +1,14562 @@
+{
```

Comment:
I guess it's not intentional, this project uses yarn.
If not, we should consider this kinda change in a different PR with other related updates. I would prefer to have a test setup before applying this change tho :)

Reply:
Ah, yes, have missed this file, will remove. 👍